mifosng-android/src/main/res/values/strings.xml

```diff
@@ -44,6 +44,7 @@
     <string name="loanterm_month">Select Loan Term</string>
     <string name="nominal">Nominal annual intrest </string>
     <string name="submittedon">Submitted On</string>
+    <string name="disbursementon">Disbursement On</string>
```

Comment:
@mdew192837 Community app use "Disbursement On" not "Disbursement Date" at this place. Also using the same phrases in many places it is a bad idea. Phrases same in English can be translated differently in other languages.